### DISCUSSION TITLE
Guidance on version numbers in installation directory paths?

### DISCUSSION THREAD

**Discussion by cvrebert** [microsoft/winget-pkgs#122991] [Q&A]
Is there any policy/convention/style guidance for the directories which winget packages install into?

E.g. The installer for one program that I'm looking at writing a manifest for installs into `C:\Program Files\fooproduct 123.4` by default. So the version is incorporated into the path.

Should I just leave the path as-is, per the upstream vanilla installer? Should I remove the version number (`C:\Program Files\fooproduct` )? Should I make the version portion number a subdirectory (`C:\Program Files\fooproduct\123.4` )? Or something else?

**Comment by mdanish-kh** [comment]
WinGet simply invokes the installer with silent (and/or additional arguments specified in the manifest), and then it's up to the installer to install wherever it wants. Unless the installer supports a CLI switch to change install location (which can then be specified in the manifest to support `--location` option for winget), WinGet does not have/enforce any control over install locations.
